src/glib/mine/svm.h

```diff
@@ -13,376 +13,640 @@
 #include "libsvm.h"
 
 namespace TSvm {
+  
+class TSvmParam{
```

Comment:
This class seams to serve two purposes. One is to store parameters used for training and another to store parameters used when predicting. I assume this is just one-on-one with that libsvm expects. If that is the case, we should call them `TLibSvmParma` to make this explicit.